src/blog/posts/forward-to-2020.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Looking forward to 2020
+published: true
+description: Moving Heaps Good Dev from PoC to MVP.
+tags: ["post", "meta"]
+slug: forward-to-2020
+date: 2020-02-23
+authors: ["scottcabot"]
+---
+
+## Looking Back
+
+In 2019 we launched Heaps Good Dev with the intent to bring together and grow the South Australian development community. We were really happy with everyone's eagerness to jump on board and help grow Heaps Good Dev. The response in our Slack, by members of public, the event organisers and event sponsors, has been extremely exciting for us.
+
+With that being said 2019 was very much a Proof of Concept for Heaps Good Dev. A concept we feel worked well but we would love to iterate on this year.
+
+## Looking Forward
+
+In 2020 we would like to take our Proof of Concept and turn it into a Minimum Viable Product. We have observed what our community seems to be receptive to and also what we think we can provide outside of everyone's regularly scheduled meetups. We look forward to trying different event formats this year while also finding new ways to engage with our community and bring them more exposure across Adelaide.
+
+We have already taken some steps to help make this possible.
+
+### Website Launch
+
+If you're reading this then you might have noticed the Heaps Good Dev website has had a major refresh. We hope to add regular content to the blog, especially if we can highlight local developers articles, talks and events. A big thanks to Dan Harris for putting together the blog and to Jakob Pennington for automating everything (because isn't that what tech is about?).
+
+### Heaps Good Dev Meetups
+
+The Heaps Good Dev Beers meetup events last year were supposed to counter more of structured talk based events to give members of our community an opportunity to engage with one another socially. While we received positive feedback about the events we noticed we could do more to be more inclusive to our members.
+
+Moving forward we will be dropping the "Beers" and simply renaming to the Heaps Good Dev meetup. Our focus will be on events that do not revolve around alcohol exclusively, and will also be held at non-pub venues. We would still like to have a social event on our calendar but in the short term we want a first time experience at a Heaps Good Dev event to be one that is welcome to all.
```

Comment:
Not sure if we need to re-phrase this. Will discuss 🙂

Reply:
Haven't heard back about this on Slack but we can tweak it once it's in dev.